src/routes/app/settings/teams/[id]/+page.server.ts

```diff
@@ -249,4 +258,26 @@ export const actions: Actions = {
       },
     }
   },
+  updateTeamModel: async ({ request, params, locals }) => {
+    const teamId = Number(params.id)
+    const fields = Object.fromEntries(await request.formData())
+    const model = fields.model.toString()
+
+    const requestingUserId = locals.currentUser.id
+    if (!(await isUserAdmin(teamId, requestingUserId))) {
+      return fail(401, {
+        modelSection: {
+          error: 'You are no admin of this team.',
```

Comment:
```suggestion
          error: 'You are not admin of this team.',
```

Reply:
Nice catch! I'll fix all occurrences of this :)